quadrat/child-theme.json

```diff
@@ -349,7 +349,7 @@
 			},
 			"core/query-pagination": {
 				"typography": {
-					"fontSize": "18px",
+					"fontSize": "20px",
```

Comment:
This should use the variable var(--wp--preset--font-size--normal)

Reply:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b